runtime/src/accounts_db.rs

```diff
@@ -1199,11 +1199,13 @@ impl RecycleStores {
     }
 
     fn add_entries(&mut self, new_entries: SnapshotStorage) {
-        self.total_bytes += new_entries.iter().map(|e| e.total_bytes()).sum::<u64>();
+        let mut total_bytes = 0;
```

Comment:
not a big fan of this name; it's not the total number of bytes, it's the bytes in the new entries.

I would be happier with either:
1. renaming this: `new_bytes`
2. just taking a mutable reference to `self.total_bytes` and accumulating into that.

Reply:
done